doc/source/reference/random/bit_generators/index.rst

```diff
@@ -35,14 +33,18 @@ The included BitGenerators are:
 .. _`Random123`: https://www.deshawresearch.com/resources_random123.html
 .. _`SFC author's page`: http://pracrand.sourceforge.net/RNG_engines.txt
 
+.. autosummary::
+   :toctree: generated/
+
+    BitGenerator
+
 .. toctree::
    :maxdepth: 1
 
-   BitGenerator <bitgenerators>
-   MT19937 <mt19937>
-   PCG64 <pcg64>
-   Philox <philox>
-   SFC64 <sfc64>
+    MT19937 <mt19937>
```

Comment:
Does 3 or 4 spaces matter?  File seem to use both.

Reply:
unified to 4, would be nice to do this gradually across the documentation

Reply:
I think it only has to be consistent per file to work. Pretty sure the default is 3 spaces everywhere.